install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1603,6 +1603,55 @@ hubble:
       #    hosts:
       #      - chart-example.local
 
+  # -- Hubble flows export.
+  export:
+    # --- Defines max file size of output file before it gets rotated.
+    fileMaxSizeMb: 10
+    # --- Defines max number of backup/rotated files.
+    fileMaxBackups: 5
+    # --- Static exporter configuration.
+    # Static exporter is bound to agent lifecycle.
+    static:
+      enabled: false
+      filePath: /var/run/cilium/hubble/events.log
+      fieldMask: []
+      # - time
+      # - source
+      # - destination
+      # - verdict
+      allowList: []
+      # - '{"verdict":["DROPPED","ERROR"]}'
+      denyList: []
+      # - '{"source_pod":["kube-system/"]}'
+      # - '{"destination_pod":["kube-system/"]}'
+    # --- Dynamic exporters configuration.
+    # Dynamic exporters may be reconfigured without a need of agent restarts.
+    dynamic:
+      enabled: false
+      config:
+        # ---- Name of configmap with configuration that may be altered to reconfigure exporters within a running agents.
+        mapName: cilium-flowlog-config
+        # ---- True if helm installer should create config map.
+        # Switch to false if you want to self maintain the file content.
+        createMap: true
+        # ---- Exporters configuration in YAML format.
+        content: |
```

Comment:
I hadn't considered supplying a default config, but I this this is reasonable. 

I do wonder if it should be a string, or if we should set it as YAML and then use `toYaml` on on it. Do you have a specific preference @marqc? What about you @michi-covalent ?

Reply:
Actually, the more I think of it, I think we should have this be an object instead of a string. This would make it easier to override options and merge them with multiple values.yaml

Reply:
I changed it to yaml